### PR TITLE
metrics: add qps metrics and remove unnecessary metrics

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

add `Store write qps` and `Store read qps`

remove `Total written bytes on hot peer Regions`  and `Total read bytes on hot peer Regions`

image

image

### Check List

Tests
- No code

### Release note

```release-note
None.
```

### CODE REVIEW

File: metrics/grafana/pd.json [hunk 1]
```diff
@@ -4712,7 +4699,7 @@
           "timeFrom": null,
           "timeRegions": [],
           "timeShift": null,
-          "title": "Store write rate keys",
+          "title": "Store write qps",
```
Comment (rleungx) [inline metrics/grafana/pd.json:4702]: qps may be not a good name since the unit is ops.

File: metrics/grafana/pd.json [hunk 2]
```diff
@@ -5944,26 +5920,19 @@
...
-              "legendFormat": "store-{{store}}-in",
-              "refId": "B"
             }
           ],
           "thresholds": [],
           "timeFrom": null,
           "timeRegions": [],
           "timeShift": null,
-          "title": "Direction of hotspot transfer leader",
+          "title": "Hot cache Read entry numbers",
```
Comment (rleungx) [inline metrics/grafana/pd.json:5935]:
```suggestion
          "title": "Hot cache read entry numbers",
```

**Review by Yisaer — COMMENTED** [review]
I will give LGTM when you address the comment

File: metrics/grafana/pd.json (hunk 1 above)
Comment (lhy1024) [reply metrics/grafana/pd.json:4702]: how about `Store write query`?

Comment (lhy1024) [reply metrics/grafana/pd.json:4702]: how about `Store write query`?

Comment (rleungx) [reply metrics/grafana/pd.json:4702]: better than qps

**Review by bufferflies — APPROVED** [review]
lgtm

**Comment by lhy1024** [comment]
@rleungx @Yisaer  PTAL

File: metrics/grafana/pd.json [hunk 3]
```diff
@@ -4279,16 +4264,16 @@
           "dashLength": 10,
           "dashes": false,
           "datasource": "${DS_TEST-CLUSTER}",
-          "decimals": 2,
-          "description": "The total bytes of hot write on leader Regions for each TiKV instance",
+          "decimals": 0,
```
Comment (rleungx) [inline metrics/grafana/pd.json:4267]: Why change it?

File: metrics/grafana/pd.json [hunk 4]
```diff
@@ -4443,8 +4428,8 @@
           },
           "yaxes": [
             {
-              "decimals": 0,
-              "format": "short",
+              "decimals": null,
```
Comment (rleungx) [inline metrics/grafana/pd.json:4431]: ditto

**Review by rleungx — APPROVED** [review]

**Pull request merged**